subnets.tf

```diff
@@ -45,3 +45,22 @@ resource "aws_internet_gateway" "assessment" {
   vpc_id = aws_vpc.assessment.id
   tags   = var.tags
 }
+
+#-------------------------------------------------------------------------------
+# Create a NAT gateway for the private subnets.
+# -------------------------------------------------------------------------------
+resource "aws_eip" "nat_gw" {
+  provider = aws.provisionassessment
+
+  tags = var.tags
+  vpc  = true
+}
+
+resource "aws_nat_gateway" "nat_gw" {
```

Comment:
Should we think about including the Guacamole Docker images as part of the AMI so that we can re-remove this gateway?

Reply:
That is not a bad idea, but I have a sense of deja vu.  I think we discussed this before but decided against it, maybe because this was back when we were first cloudifying CyHy and making our AMIs public, and hence we wanted to keep sensitive data out of them.

@dav3r and @felddy, do you remember when we discussed this before?

Reply:
I don't remember the details of this discussion from the past.  I am definitely in favor of baking the Guacamole Docker image into our AMI if it means we can get rid of the NAT gateway and related port 443 access rules.

Reply:
OK.  I created cisagov/cool-system#158 for this change.